app/admin/models/Orders_model.php

```diff
@@ -154,9 +158,25 @@ public function scopeListFrontEnd($query, $options = [])
             }
         }
 
+        $search = trim($search);
+        if (strlen($search)) {
+            $query->search($search, $searchableFields);
+        }
+
+        if (($startDateTime = array_get($dateTimeFilter, 'orderDateTime.start', false)) AND ($endDateTime = array_get($dateTimeFilter, 'orderDateTime.end', false))) {
+            $query = $this->scopeWhereBetweenOrderDateTime($query, Carbon::parse($startDateTime)->format('Y-m-d H:i:s'), Carbon::parse($endDateTime)->format('Y-m-d H:i:s'));
+        }
+
         return $query->paginate($pageLimit, $page);
     }
 
+    public function scopeWhereBetweenOrderDateTime($query, $start, $end)
```

Comment:
You missed the other comment to rename `orderDateTime.start` and `orderDateTime.end` to `orderDateTime.startAt` and `orderDateTime.endAt`

Reply:
I did indeed - updated